subprojects/docs/src/docs/userguide/dep-man/03-controlling-transitive-dependencies/platforms.adoc

```diff
@@ -228,6 +247,11 @@ Supported members of a version declaration are:
 
 Dependency declaration can either be declared as a simple string, in which case they are interpreted as `group:artifact:version` coordinates, or separating the version declaration from the group and name:
 
+[NOTE]
+====
+For aliases same rules apply as described in <<platforms.adoc#sub:mapping-aliases-to-accessors, aliases and their mapping to type safe accessors>> section.
```

Comment:
💄 : For aliases, the rules described in the section <<platforms.adoc#sub:mapping-aliases-to-accessors, aliases and their mapping to type safe accessors>> apply as well.